### ISSUE TITLE
Is it normal during training for getting artifacts on the generation results?

### ISSUE THREAD

**Issue by shileims** [advimman/lama#218]
Hi,
     Thanks for your really great work. 
     I am trying to finetune the model and use the same mask generation strategy on the places dataset.
     But I found the intermediate-generation results have severe artifacts problems. 
     I would like to ask do you think it is normal? Is it overfitting or underfitting? Thanks
![image](https://user-images.githubusercontent.com/104052591/221393872-f15805a3-a682-4b83-99fa-4878d6ed422e.png)

**Comment by kign17019999** [comment]
hi. Im interested about finetune the model by using my owndataset as you did.
can you suggest me what should i do too because I am very new to this field.
thankyou @shileims
